cfgov/jinja2/v1/_includes/molecules/related-metadata.html

```diff
@@ -135,13 +140,9 @@ <h3 class="h4">
    ========================================================================== #}
 
 {% macro _text(text) %}
-
-<p class="u-mb0">
```

Comment:
The wagtail RTE adds a paragraph, so looking at the generated markup it was looking like:
```
<p class="u-mb0"></p>
<p>Some wagtail entered text</p>
```

This could be changed to a div, but I didn't see how the `u-mb0` was used as removing it made no apparent visual difference.